tools/c7n_azure/c7n_azure/query.py

```diff
@@ -197,4 +207,25 @@ def register_actions_and_filters(registry, _):
             klass.action_registry.register('notify', Notify)
 
 
+@six.add_metaclass(QueryMeta)
+class ChildResourceManager(QueryResourceManager):
+
+    ParentSpec = namedtuple("ParentSpec", ["manager_name", "annotate_parent"])
```

Comment:
we don't use this naming scheme in general for class vars, we generally refer it to globals if not GLOBALS, i'd suggest `parent_spec`

Reply:
class name for named tuple can stay as is, per class naming conventions